x-pack/docs/en/rest-api/security/app-privileges.asciidoc

```diff
@@ -0,0 +1,285 @@
+[role="xpack"]
+[[security-api-app-privileges]]
+=== Application Privileges APIs
+
+The Application Privileges API enables you to add, remove, and retrieve
+{xpack-ref}/defining-roles.html#roles-application-priv[Application Privileges].
+
+==== Request
+
```

Comment:
Can we separate these into separate pages for the various APIs (similar to the separate pages for creating and deleting ML objects e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-apis.html)? If so, I can do that split.